### PR TITLE
QA: Update dependencies

### PR DESCRIPTION
**Pull request by michalbundyra**
|    Q          |   A
|-------------- | ------
| QA            | yes

### CODE REVIEW

File: img/laminas-components-rgb.svg [hunk 1]
```diff
@@ -1 +1,98 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (froschdesign) [inline img/laminas-components-rgb.svg:1]: This changes brings back the old and wrong aligned logo.

Comment (michalbundyra) [reply img/laminas-components-rgb.svg:1]: this is what gulp call did ... I think we haven't updated them in assets directory?

Comment (froschdesign) [reply img/laminas-components-rgb.svg:1]: Uff, maybe.

Comment (froschdesign) [reply img/laminas-components-rgb.svg]: Right: https://github.com/laminas/laminas.github.io/commit/d470e72fe863f54dd3cf1c83f5496c01881f7876

Comment (michalbundyra) [reply img/laminas-components-rgb.svg]: Reverted and updated in asset/img now

**Pull request merged**